### ISSUE TITLE
Consider changing uplugin file "Installed" setting to false and/or "EnabledByDefault" setting to true

### ISSUE THREAD

**Issue by sbseltzer** [fpwong/BlueprintAssistWiki#130]
I'm not sure if the current uplugin settings are this way intentionally, but if this is a one-license-per-user setup and not included in game builds, you may want to consider doing the same thing Riderlink does when installed as an engine plugin.

`Installed` - Setting this to false will prevent the editor from asking you to add the plugin to the uproject file plugins list.
`EnabledByDefault` - Setting this to true will prevent the need to explicitly enable the plugin for each project you wish to use it in.

Note: I'm using it in a source build of the engine, so maybe these settings don't make sense or can't be used when installing to a pre-built engine via the Epic Games Launcher? Just something to think about.

**Comment by fpwong** [comment]
Hi @sbseltzer, I have tried to include `EnabledByDefault` a while ago in the uplugin but when I submitted it to the marketplace, the line was removed on the packaged version. I suppose they don't want plugins to be always enabled. I also have `Installed` set to false on my end, I guess they want that to be true as well...?

If you'd like I can invite you to the souce repo so you don't have to build from the marketplace version every time.
